lib/screenplay_web/controllers/pa_messages_api_controller.ex

```diff
@@ -20,4 +22,15 @@ defmodule ScreenplayWeb.PaMessagesApiController do
         send_resp(conn, 500, "Could not fetch audio preview")
     end
   end
+
+  def create(conn, params) do
+    case PaMessages.create_message(params) do
+      {:ok, _} ->
+        json(conn, %{success: true})
+
+      {:error, changeset} ->
+        Logger.error("[pa_message create] #{Util.format_changeset_errors(changeset)}")
+        send_resp(conn, 500, "Could not create message")
```

Comment:
**issue:** a `500` status code implies an internal server error. we probably want something in the 400-499 range (i'd go for `422`).

there's also a great guide in the phoenix docs that covers setting up the fallback controller to render out changeset errors that i think we should do: https://hexdocs.pm/phoenix/json_and_apis.html#fallbackcontroller-and-changesetjson

Reply:
Very interesting. Thanks for the link. Took me down a nice educational rabbit hole. I think I managed to get this working for us.